index.html

```diff
@@ -1887,324 +1887,639 @@ <h2>
-          readonly attribute DOMString recipient;
-          readonly attribute DOMString phone;
-        };
-      </pre>
-      <section>
+      <p>
+        A <dfn>physical address</dfn> is composed of the following concepts.
+      </p>
+      <dl data-sort="">
+        <dt>
+          <dfn>Country</dfn>
+        </dt>
+        <dd>
+          The country corresponding to this address.
+        </dd>
+        <dt>
+          <dfn>Address line</dfn>
+        </dt>
+        <dd>
+          The most specific part of the address. It can include, for example, a
+          street name, a house number, apartment number, a rural delivery
+          route, descriptive instructions, or a post office box number.
+        </dd>
+        <dt>
+          <dfn>Region</dfn>
+        </dt>
+        <dd>
+          The top level administrative subdivision of the country. For example,
+          this can be a state, a province, an oblast, or a prefecture.
+        </dd>
+        <dt>
+          <dfn>City</dfn>
+        </dt>
+        <dd>
+          The city/town portion of the address.
+        </dd>
+        <dt>
+          <dfn>Dependent locality</dfn>
+        </dt>
+        <dd>
+          The dependent locality or sublocality within a city. For example,
+          used for neighborhoods, boroughs, districts, or UK dependent
+          localities.
+        </dd>
+        <dt>
+          <dfn>Postal code</dfn>
+        </dt>
+        <dd>
+          The postal code or ZIP code, also known as PIN code in India.
+        </dd>
+        <dt>
+          <dfn>Sorting code</dfn>
+        </dt>
+        <dd>
+          The sorting code as used in, for example, France.
+        </dd>
+        <dt>
+          <dfn>Language code</dfn>
+        </dt>
+        <dd>
+          The language in which the address is provided. It's used to determine
+          the field separators and the order of fields when formatting the
+          address for display.
+        </dd>
+        <dt>
+          <dfn>organization</dfn>
+        </dt>
+        <dd>
+          The organization, firm, company, or institution at this address.
+        </dd>
+        <dt>
+          <dfn>recipient</dfn>
+        </dt>
+        <dd>
+          The name of the recipient or contact person. This member may, under
+          certain circumstances, contain multiline information. For example, it
+          might contain "care of" information.
+        </dd>
+        <dt>
+          <dfn>phone number</dfn>
+        </dt>
+        <dd>
+          The phone number of the recipient or contact person at this address.
+        </dd>
+      </dl>
+      <section data-dfn-for="PaymentAddress" data-link-for="PaymentAddress">
         <h2>
-          Creating a <code>PaymentAddress</code>
+          <dfn>PaymentAddress</dfn> interface
         </h2>
+        <pre class="idl">
+          [SecureContext, Exposed=(Window,Worker), Constructor(optional AddressInit details)]
```

Comment:
Does exposing this in workers make sense?

Reply:
I think we want MAY want expose it for Payment Hander. I can redact it until we are 100% on the API.